packages/sessions-sdk-ts/src/index.ts

```diff
@@ -489,3 +498,102 @@ export type Session = {
   ) => Promise<TransactionResult>;
   sessionInfo: z.infer<typeof sessionInfoSchema>;
 };
+
+const TRANSFER_MESSAGE_HEADER = `Fogo Transfer:
+Signing this intent will transfer the tokens as described below.
+`;
+
+type SendTransferOptions = {
+  adapter: SessionAdapter;
+  walletPublicKey: PublicKey;
+  signMessage: (message: Uint8Array) => Promise<Uint8Array>;
+  mint: PublicKey;
+  amount: bigint;
+  recipient: PublicKey;
+};
+
+export const sendTransfer = async (options: SendTransferOptions) => {
+  const sourceAta = getAssociatedTokenAddressSync(
+    options.mint,
+    options.walletPublicKey,
+  );
+  const destinationAta = getAssociatedTokenAddressSync(
+    options.mint,
+    options.recipient,
+  );
+  const program = new IntentTransferProgram(
+    new AnchorProvider(options.adapter.connection, {} as Wallet, {}),
+  );
+  const umi = createUmi(options.adapter.connection.rpcEndpoint);
+  const metaplexMint = metaplexPublicKey(options.mint.toBase58());
+  const metadataAddress = findMetadataPda(umi, { mint: metaplexMint })[0];
+  const metadata = await safeFetchMetadata(umi, metadataAddress);
+  const symbol = metadata?.symbol ?? undefined;
+
+  return options.adapter.sendTransaction(undefined, [
+    createAssociatedTokenAccountIdempotentInstruction(
+      options.adapter.payer,
+      destinationAta,
+      options.recipient,
+      options.mint,
+    ),
+    await buildTransferIntentInstruction(program, options, symbol),
+    await program.methods
+      .sendTokens()
+      .accounts({
+        destination: destinationAta,
+        mint: options.mint,
+        source: sourceAta,
+        // eslint-disable-next-line unicorn/no-null
+        metadata: null,
```

Comment:
is it expected that metadata appears twice in this object?

Reply:
lol no, I guess I had this as `null` initially but forgot to remove it when I did the real implementation.  Good catch, will fix.